resources/translations/fi.edn

```diff
@@ -300,9 +300,8 @@
            :phase-pending "Vaihe tulossa:"
            :phase-rejected "Vaihe hylätty:"
            :rejected "Hylätty"}
-  :search {:close-search "Sulje haku"
-           :open-search "Avaa haku"
-           :search-parameters "Hakutermit"}
+  :search {:clear-search "Nollaa etsintä"
+           :search "Etsi"}
```

Comment:
Hae?

Reply:
That would confuse it with "Hae uusia resursseja"

Reply:
fair enough